camus-kafka-coders/src/main/java/com/linkedin/camus/etl/kafka/coders/JsonStringMessageDecoder.java

```diff
@@ -62,7 +63,12 @@ public void init(Properties props, String topicName) {
     String payloadString;
     JsonObject jsonObject;
 
-    payloadString = new String(payload);
```

Comment:
Only a few encodings are allowed for JSON: UTF-8, UTF-16 and UTF-32 if memory serves.
Therefore: JSON is pretty much always in UTF-8.